src/main/java/org/candlepin/resteasy/filter/SecurityHoleAuthorizationFilter.java

```diff
@@ -132,10 +134,14 @@ private String checkCloudAccountOrganization(AnonymousCloudConsumer anonymousClo
                 anonymousCloudConsumer.getCloudOfferingId());
         }
         catch (OrgForCloudAccountNotCreatedYetException e) {
-            throw new TooManyRequestsException(ORG_DOES_NOT_EXIST_AT_ALL);
+            String msg = "Upstream anonymous owner not yet created by adapters";
+            throw new TooManyRequestsException(i18nProvider.get().tr(msg))
+                .setRetryAfterTime(ORG_DOES_NOT_EXIST_AT_ALL);
         }
         catch (OrgForCloudAccountNotEntitledYetException e) {
-            throw new TooManyRequestsException(ORG_IS_NOT_ENTITLED);
+            String msg = "Upstream anonymous owner not yet entitled by adapters";
+            throw new TooManyRequestsException(i18nProvider.get().tr(msg))
```

Comment:
Same, you need to pass in the hardcoded string or else it won't get translated. The way it works is gettext scans the actual code for instances of tr(), and finds the strings passed to it as keys to translate. if a variable is passed it won't work

Reply:
Ah yeah, I forget that is a thing. Updated